docs/src/pages/guides/layout.js

```diff
@@ -0,0 +1,271 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+import Highlight from "react-highlight";
+import {
+  Box, Flex, SectionTitle, SubsectionTitle, Title, Link, List, ListItem,
+} from "@nulogy/components";
+import {
+  DocText as Text, Layout, Intro, DocSection, DocSubsection,
+} from "../../components";
+
+export default () => (
+  <Layout>
+    <Helmet>
+      <title>Layout</title>
+    </Helmet>
+    <Box pt="x4" mb="x6">
+      <Title m="none">Layout</Title>
+      <Intro>This article provides a breakdown of basic concepts and directions on how to compose the layout of any complexity.</Intro>
+    </Box>
+    <Flex mb="x8" flexDirection={ { small: "column", medium: "row", large: "row" } }>
+      <Flex bg="whiteGrey" justifyContent="center" alignItems="center" p="x3" width={ { small: 1, medium: 2 / 3, large: 1 / 2 } } height={ { small: "200px", medium: "auto", large: "auto" } }>
+        <Box bg="darkBlue" height="80px" width="80px" mr="x3" />
+        <Box bg="darkBlue" height="80px" width="80px" mr="x3" />
+        <Box bg="darkBlue" height="80px" width="80px" />
+      </Flex>
+      <Flex bg="lightGrey" flexDirection="column" justifyContent="center" alignItems="center" p="x3" width={ { small: 1, medium: 1 / 3, large: 1 / 2 } } height={ { small: "200px", medium: "auto", large: "auto" } }>
+        <Box height="80px" width="80px" mb="x3" bg="blackBlue" />
+        <Box height="80px" width="80px" mb="x3" bg="darkBlue" />
+        <Box height="80px" width="80px" bg="blue" />
+      </Flex>
+    </Flex>
+
+    <DocSection>
+      <SectionTitle>Building blocks</SectionTitle>
+      <Text><Link href="/components/box">Box</Link> and <Link href="/components/flex">Flex</Link> components are basic building blocks of any layout composition and are based on CSS box model and CSS flexbox layout model. Listed below are the links that will help you familiarize yourself with these concepts and help you be effective at using Box and Flex components.</Text>
+      <List mb="x3">
+        <ListItem><Link href="https://www.w3schools.com/css/css_boxmodel.asp">CSS box model</Link></ListItem>
+        <ListItem><Link href="https://css-tricks.com/snippets/css/a-guide-to-flexbox/">CSS flexbox layout model</Link></ListItem>
+        <ListItem><Link href="https://flexboxfroggy.com/">Flexbox Froggy</Link></ListItem>
+      </List>
+    </DocSection>
+
+    <DocSection>
+      <SectionTitle>Spacing</SectionTitle>
+      <SubsectionTitle>Padding and Margins</SubsectionTitle>
+      <Text>Padding is used for adding spacing inside of the component and margin is used for adding space around the component.</Text>
+      <Flex bg="lightGrey" style={ { display: "inline-flex" } }>
+        <Box><Flex color="white" bg="blackBlue" m="x3" p="x3">Box</Flex></Box>
```

Comment:
We could label these boxes to make it even more immediately obvious. e.g instead of saying Box inside each box, we could say something like Box with padding, Box with margin, Box with both